website/content/userguide/minio.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "setting up MinIO on gokrazy"
+weight: 50
+---
+
+# Setting up MinIO on gokrazy
+
+Since MinIO is a storage solution, one needs to have the permanent storage with
+a filesystem that the linux kernel understands configured. Further instructions
+on how to do this can be found in the [Quickstart](/quickstart/) article or the
+`gokr-packer` output.
+
+This article also assumes that you have an instance directory set up.
+(also see [Quickstart](/quickstart/))
+
+## Configuring the ENV Vars and flags
+
+You can find a detailed description on how to set the flags and environment vars
+in the article [per-package configuration](/userguide/package-config/).
+
+`${INSTANCE?}/env/github.com/minio/minio/env.txt`
+```env
+MINIO_ROOT_USER=minio
+MINIO_ROOT_PASSWORD=minio-on-gokrazy
+HOME=/perm/minio
```

Comment:
The HOME= line should not be necessary. gokrazy sets `HOME` automatically. See also https://gokrazy.org/userguide/process-interface/ :)

Can you remove the HOME= line please?

Reply:
Yeah this was due to me using `/perm/minio_data` as foldername ...